### PR TITLE
update lib doc

### PR DESCRIPTION
**Pull request by JoeyHwong-gk**
Signed-off-by: JoeyHwong <joeyhwong@gknow.cn>

### CODE REVIEW

File: lib/sedna/algorithms/multi_task_learning/__init__.py [hunk 1]
```diff
@@ -12,5 +12,5 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-from .task_jobs import *
+import task_jobs
```
Comment (JensenJian) [inline lib/sedna/algorithms/multi_task_learning/__init__.py:15]: For consistency, why we are not using from . import task_jobs like the other files did

Comment (JoeyHwong-gk) [reply lib/sedna/algorithms/multi_task_learning/__init__.py:15]: fix

File: lib/sedna/README.md [hunk 2]
```diff
@@ -1,2 +1,224 @@
...
+   
+    # get hard exmaple mining algorithm from config
+    hard_example_mining = IncrementalLearning.get_hem_algorithm_from_config(
+        threshold_img=0.9
+    )
+   
+    # create Incremental Learning infernece instance
+    il_job = IncrementalLearning(
+        estimator=Estimator,
+        hard_example_mining=hard_example_mining,
```
Comment (luosiqi) [inline lib/sedna/README.md:149]: Is there a redundant comma after "hard_example_mining"?

File: lib/sedna/README.md [hunk 3]
```diff
@@ -1,2 +1,224 @@
...
+	- `img_rgb` is the sample used to inference
+    - `deal_infer_rsl` is a function used to process result after model predict
+    - `input_shape` is the parameters of `Estimator` in inference
+    - `results` is the result predicted by model
+    - `final_res` is the result after process by `deal_infer_rsl`
+    - `is_hard_example` tells if the sample is hard sample or not
+
+## Customize algorithm
+
+Sedna provide a class called `class_factory.py` in `common` package, only a few lines of changes are required to become a module of sedna.
```
Comment (luosiqi) [inline lib/sedna/README.md:187]: 1、provide -> provides
2、, in which only a few lines of changes...

Comment (JoeyHwong-gk) [reply lib/sedna/README.md:192]: check

File: lib/sedna/README.md (hunk 2 above)
Comment (JoeyHwong-gk) [reply lib/sedna/README.md:154]: check

**Pull request merged**